apps/meteor/server/services/upload/service.ts

```diff
@@ -8,9 +8,11 @@ import { FileUpload } from '../../../app/file-upload/server';
 export class UploadService extends ServiceClassInternal implements IUploadService {
 	protected name = 'upload';
 
-	async uploadFile({ buffer, details }: IUploadFileParams): Promise<IUpload> {
-		const fileStore = FileUpload.getStore('Uploads');
-		return fileStore.insert(details, buffer);
+	async uploadFile({ buffer, details, userId }: IUploadFileParams): Promise<IUpload> {
+		return Meteor.runAsUser(userId, () => {
```

Comment:
is this required? I recall something going on with `fileStore.insert`, but having this here seems weird

Reply:
Yup. Required, otherwise, throws error